src/extension/options-page/DashboardDialogs/Wallet.tsx

```diff
@@ -261,6 +269,14 @@ export function DashboardWalletCreateDialog(props: WrappedDialogProps<object>) {
                 children: (
                     <div>
                         <TextField
+                            helperText={
+                                name.length > 32
+                                    ? t('input_length_exceed_prompt', {
+                                          name: t('wallet_name').toLowerCase(),
+                                          length: 32,
```

Comment:
```suggestion
                                          length: 32,
```

extract max length to constants file.